tests/functional/sources/test_simple_source.py

```diff
@@ -0,0 +1,211 @@
+import os
+import pytest
+import yaml
+from dbt.exceptions import ParsingException
+
+from dbt.tests.util import run_dbt, update_config_file
+from dbt.tests.tables import TableComparison
+from tests.functional.sources.common_source_setup import (
+    BaseSourcesTest,
+)
+from tests.functional.sources.fixtures import (
+    macros__macro_sql,
+    malformed_models__schema_yml,
+    malformed_models__descendant_model_sql,
+    malformed_schema_tests__schema_yml,
+    malformed_schema_tests__model_sql,
+)
+
+
+class SuccessfulSourcesTest(BaseSourcesTest):
+    @pytest.fixture(scope="class", autouse=True)
+    def setUp(self, project):
+        self.run_dbt_with_vars(project, ["seed"])
+        os.environ["DBT_ENV_CUSTOM_ENV_key"] = "value"
+
+        yield
+
+        del os.environ["DBT_ENV_CUSTOM_ENV_key"]
+
+    @pytest.fixture(scope="class")
+    def macros(self):
+        return {"macro.sql": macros__macro_sql}
+
+    def _create_schemas(self, project):
+        schema = self.alternative_schema(project.test_schema)
+        project.run_sql(f"drop schema if exists {schema} cascade")
+        project.run_sql(f"create schema {schema}")
+
+    def alternative_schema(self, test_schema):
+        return test_schema + "_other"
+
+    @pytest.fixture(scope="class", autouse=True)
```

Comment:
Does the order of execution stay the same for this and the setUp fixture? Or doesn't it matter?

Reply:
In this case the order doesn't matter.  I think if order does matter the clearest way to enforce (and communicate) the order is to request the first fixture in the second fixture.
```
@pytest.fixture(scope="class", autouse=True)
def setUp(self, project):
    ...
@pytest.fixture(scope="class", autouse=True)
def createDummyTables(self, setUp, project):
    ...
```